df.units.xml

```diff
@@ -1892,8 +1892,13 @@
                     <flag-bit name='precise'/>
                     <flag-bit name='charge'/>
                     <flag-bit comment='multi-attack'/>
+                    <flag-bit/>
+                    <flag-bit name='lightly_tap'/>
+                    <flag-bit/>
+                    <flag-bit/>
+                    <flag-bit name='spar_report'/>
                 </bitfield>
-                <int16_t name='unk_38'/>
+                <int16_t name='attack_skill'/>
```

Comment:
That it should.

Reply:
For the record, you don't strictly *need* the base-type there, since it's the same as the base-type of the job_skill enum itself, but it's good to have for clarity sometimes.